### PR TITLE
Created examples list in commands.py to auto update readme, updated contributing.md to reflect this

### PR DESCRIPTION
**Pull request by ChaseBosman**
Wanting to push this change before anyone else gets a chance to add more modules and not update the examples list. Planning to implement the readme auto-updater in the next few days.

### CODE REVIEW

File: commands.py [hunk 1]
```diff
@@ -39,6 +39,30 @@ def __init__(self):
...
+    def get_examples():
+        # examples is a list containing strings in the form "!(command) (Optional Argument) - (Description)"
+        examples = [
+            "!urban word - Searches and returns urban dictionary for provided word",
+            "!wiki word/phrase - Searches and returns urban dictionary for provided word/phrase",
+            "!roll 5d50 - Provides randomized rolls of a 5 sided dice 50 times",
+            "!flip - Provides randomized head or tails",
+            "!joke - Provides a random joke",
+            "!chuck - Provides a random Chuck Norris joke",
+            "!quote - Provides a random quote",
```
Comment (svimanet) [inline commands.py:52]:
``` suggestion
            "!quote - Returns quote of the day from an API",
```

Comment (svimanet) [inline commands.py:52]:
```suggestion
            "!quote - Returns a random quote",
```

File: commands.py [hunk 2]
```diff
@@ -39,6 +39,30 @@ def __init__(self):
...
+            "!horoscope sign - Provides a daily horoscope corresponding to the provided sign",
+            "!cat - Provides a link to a super cute cat pic",
+            "!dog - Provides a link to a super cute dog pic",
+            "!jesus - Provides a special message from jesus",
+            "!catfact - Provides a random cat fact",
+            "!draw - Provides a card randomly drawn from a deck",
+            "!help - Displays available commands",
+            "!covidglobal - Provides current covid statistics globally",
+            "!covidcountry Germany - Provides covid statistics for Germany",
+            "!sentiment word/phrase- Provides if the word/phrase is a sentiment or not"
```
Comment (svimanet) [inline commands.py]: Some of the descriptions had errors, and (IMO) "returns" makes more sense in this context.
Suggested some changes.

```suggestion
            "!urban word - Searches and returns urban dictionary description of a specified term.",
            "!wiki word/phrase - Searches and returns wikipedia description of a specified term",
            "!roll 5d50 - Returns randomized rolls of a 5 sided dice 50 times. Any format of xDy works.",
            "!flip - Flips a coin and returns heads, tails, or ... edge?",
            "!joke - Returns a random joke.",
            "!chuck - Returns a random Chuck Norris joke.",
            "!quote - Returns the Quote of the Day. Only changes once per day.",
            "!horoscope libra - Returns a daily horoscope corresponding to the provided zodiac sign.",
            "!cat - Returns a link to a super cute cat pic.",
            "!dog - Returns a link to a super cute dog pic.",
            "!jesus - Returns a special message from our lord and saviour, Jesus Christ",
            "!catfact - Returns a random cat fact.",
            "!draw - Returns a random card from a standard 52 card deck.",
            "!help - Displays available commands.",
            "!covidglobal - Returns current covid-19 statistics globally.",
            "!covidcountry Germany - Returns covid-19 statistics for the specified country.",
            "!sentiment word/phrase - Returns the sentiment value of a specified sentence, based on an API."
```

**Review by svimanet — COMMENTED** [review]
Though I don't think unfinished code should ever be committed master, I will allow this after the proposed changes. The added code does not break anything, and I have faith that you are woking on completing it.

**Comment by ChaseBosman** [comment]
> 
> 
> Though I don't think unfinished code should ever be committed master, I will allow this after the proposed changes. The added code does not break anything, and I have faith that you are woking on completing it.

Committed those suggested changes. Ill start working on the auto updater part today. While I understand the comments regarding unfinished code, would there be a better practice that I should have followed? It kind of just seems like a drawback to the open source paradigm in terms of the possibility of someone building a feature while another person may be building a feature that conflicts with it...which I guess is what makes planning just that much more important...

**Comment by svimanet** [comment]
> 
> 
> > Though I don't think unfinished code should ever be committed master, I will allow this after the proposed changes. The added code does not break anything, and I have faith that you are woking on completing it.
> 
> Committed those suggested changes. Ill start working on the auto updater part today. While I understand the comments regarding unfinished code, would there be a better practice that I should have followed? It kind of just seems like a drawback to the open source paradigm in terms of the possibility of someone building a feature while another person may be building a feature that conflicts with it...which I guess is what makes planning just that much more important...

Ideally the unfinished code would live in a separate branch until completion. In this case that _could_ be your fork. To avoid conflicts you should regularly pull master into your branch, to stay up to date. But since this task would require much more work than previous contributions, it would be a very large PR. In normal circumstances large PRs is not an issue, but this repository is aimed towards beginners and hacktoberfest participants, so yours is a special case. _And_ I'm not an experienced maintainer :)

**Pull request merged**